### DISCUSSION TITLE
What pins can I use with the 16u2 (mega)

### DISCUSSION THREAD

**Discussion by spookybear0** [NicoHood/HoodLoader2#96] [Q&A]
I have read the wiki and read all the issues and can't figure out what pins I can use on an arduino mega with HootLoader2 installed, I tried the normal digital pins, but they do not work with HootLoader2 installed, I am assuming that is because I can only use the 16u2 pins, but I am having trouble finding where they are and what number it uses for me to access it. Any help would be appreciated, Thank you.

**Comment by NicoHood** [comment]
Maybe this helps?
https://github.com/NicoHood/HoodLoader2/wiki/Arduino-Uno-Mega-16u2-Pinout
https://github.com/NicoHood/HoodLoader2/wiki/4-Pin-Header

You only have those 6+4 pins (of which 2 are gnd and vcc). It is possible that your board does not even has a 16u2, if it is a china clone.
